From 68dc7081f66d229c1bfc283faaa6479fbb72909b Mon Sep 17 00:00:00 2001
From: Wang ShaoBo <bobo.shaobowang@huawei.com>
Date: Thu, 8 Dec 2022 21:45:12 +0800
Subject: [PATCH] arm64/mpam: Fix indent format error in resctrl_parse_param()
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 68dc7081f66d229c1bfc283faaa6479fbb72909b
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I61CPK
CVE: NA

--------------------------------

This fixes indent format error in resctrl_parse_param():
 fs/resctrlfs.c:616 resctrl_parse_param() warn: ignoring unreachable code.
 fs/resctrlfs.c:616 resctrl_parse_param() warn: inconsistent indenting
 fs/resctrlfs.c:619 resctrl_parse_param() warn: inconsistent indenting

Fixes: 100e2317e9b9 ("arm64/mpam: Use fs_context to parse mount options")
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/resctrlfs.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/fs/resctrlfs.c b/fs/resctrlfs.c
index 8956237de47f..c0a84f40dcc0 100644
--- a/fs/resctrlfs.c
+++ b/fs/resctrlfs.c
@@ -622,12 +622,11 @@ static int resctrl_parse_param(struct fs_context *fc, struct fs_parameter *param
 	case Opt_caPrio:
 		ctx->enable_caPrio = true;
 		return 0;
+	default:
+		break;
+	}
 
-	return 0;
-}
-
-return -EINVAL;
-
+	return -EINVAL;
 }
 
 static void resctrl_fs_context_free(struct fs_context *fc)
-- 
2.33.0

